### DISCUSSION TITLE
how many passphrase are need after keyring is locked

### DISCUSSION THREAD

**Discussion by justinbread** [Chia-Network/chia-blockchain#10918] [Support]
after upgrated the new 1.3x, keyring is locked, i`v try many times with my key passprase, its just doesnt work, i dont know how many passphrase are need ?the whole 24?or maybe it is  my keybord-problem?

**Comment by loppefaaret** [comment]
the keyring data is stored in `C:\Users\<USERNAME>\.chia_keys\` folder - if you got issues with the passphrase, you could move out this folder, and re-import you key with your 24 word mnemonic

**Reply by justinbread** [reply]
THX BRO! HELP ME A LOT!
